src/ui/templates/index.njk

```diff
@@ -45,14 +45,20 @@
 
     {% include "partials/footer.njk" %}
 
-    <script src="/assets/js/govukFrontend.js" type="text/javascript"></script>
-    <script src="/assets/js/cookies.js" type="text/javascript"></script>
-    <script src="/assets/js/formSubmission.js" type="text/javascript"></script>
 
-    {% if cookieConsent %}
-      <script async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsId }}"></script>
-      <script>window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', '{{ googleAnalyticsId }}');</script>
-    {% endif %}
+      {%block scripts %}
+      <script src="/assets/js/jsEnabled.js" type="text/javascript" integrity="sha512-LUW2MMgLIyJzPPvhODhZMciLaqLNgBGrSMMj6zOJsH+7uSahGknt3c+VSzyUYlSnngNXZwvvNA7kc+Zvx0Mh9A==" crossorigin="anonymous"></script>
+      <script src="/assets/js/govukFrontend.js" type="text/javascript" integrity="sha512-EdyNYxz0W40gqYt1R48s+ye7mW0p7SwUTnkyUcMJ0eGRsucWodRmVpC5+QAtUBZSYeOMeWumRLnqJZ2zzkDRqg==" crossorigin="anonymous"></script>
+      <script src="/assets/js/formSubmission.js" type="text/javascript" integrity="sha512-uyTmQxJNGlnB71lDj1CCOUhKJLch3djXW1Av0BOT4g7K7riFXsDbyUmhNkhUrtpbdFICkh3lFxmviTlOKNTh7w==" crossorigin="anonymous"></script>
+      <script src="/assets/js/mojFrontend.js" type="text/javascript" integrity="sha512-Ou5NaBw3i2mTX9wG4T5qe8iVVv6Y/67xtvY1ByVY3fYf82Mc2vLGaXfs2z89f9IJ6GCKC64Cv+oRKKYWszDi5Q==" crossorigin="anonymous"></script>
+      <script src="/assets/js/cookies.js" type="text/javascript" integrity="sha512-1MU/q7OlnUX2SMKAZWls7I/EfPiitbX+ym3Dzj+f5r4qI+nQKF3Q6LZk3LRk3cqO726dQBlwhhEcJW455LM69w==" crossorigin="anonymous"></script>
```

Comment:
Could we create constants for the integrity values and then pass to the template via `corePageVariables`? https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/helpers/page-variables/core/index.ts#L55

We use this for all generic strings etc

Reply:
Have added across all the pages apart from `index.njk` as it appears it does not call `corePageVariables`.
Would be good to look into it on Monday as it will also help resolve `og:*` meta values population.